### PR TITLE
Basic Backend setup with Pontsun

### PR DESCRIPTION
**Pull request by thetractor**

### CODE REVIEW

File: config/.env.dist [hunk 1]
```diff
@@ -0,0 +1,13 @@
+####################
+# BACKEND SETTINGS #
+####################
+DEBUG=True
+SECRET_KEY='aa6hl(enzbma+(ho=z3la3uhc-x!h%-$h@rll=d*izpo23fyuv'
```
Comment (21adrian1996) [inline config/.env.dist]: Is it really best practice to check in a default key to git?
Not sure if this will eventually lead to security issues.

**Comment by schtibe** [comment]
Something strange here: why does this PR incorporate Ingvi's changes?

**Comment by thetractor** [comment]
> Something strange here: why does this PR incorporate Ingvi's changes?

Because I rebased it with master and later created a develop branch. I think the problem is that a created the develop branch from my feature branch and not from master.

File: .gitignore [hunk 2]
```diff
@@ -1 +1,11 @@
-node_modules
+##### BASE IGNORES #####
+.idea/
+*.env
+
+##### FORINTEND IGNORES #####
```
[the remark this answers is not in the document]
Comment (thetractor) [reply .gitignore]: Done with next push

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,20 @@
+FROM python:3.7.4-buster
+
+ARG USER_ID=1000
+ARG GROUP_ID=1000
+ENV PYTHONUNBUFFERED 1
```
[the remark this answers is not in the document]
Comment (thetractor) [reply Dockerfile]: We need this as this Dockerfile is for local development and not for production build

File: config/.env.dist (hunk 1 above)
Comment (thetractor) [reply config/.env.dist]: Done with next push

**Comment by thetractor** [comment]
> OK so you now take Ingvi's changes on master along with you into develop...
> Why do we even have develop branch? Also, we should change the default branch if we decide to have develop, because half of the PRs are now into master, the other half into develop...

@schtibe I changed the merge request to go directly into master

File: config/.env.dist [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+DEBUG=True
+SECRET_KEY='unsafesecretkey'
+ALLOWED_HOSTS=localhost, 127.0.0.1
+DATABASE_PASSWORD=phae0ooZee0d
+DATABASE_URL=postgres://postgres:$DATABASE_PASSWORD@localhost:5433/postgres
+STATIC_ROOT=/tmp/static/
+STATIC_URL=/static/
+MEDIA_ROOT=/tmp/static/media
+MEDIA_URL=/media/
+LOCALE_PATHS=locale/'
```
Comment (21adrian1996) [inline config/.env.dist]: There seems to be a ' at the end of the file

**Pull request merged**